src/main/java/org/jahia/modules/sitemap/utils/ConversionUtils.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * ==========================================================================================
+ * =                            JAHIA'S ENTERPRISE DISTRIBUTION                             =
+ * ==========================================================================================
+ *
+ *                                  http://www.jahia.com
+ *
+ * JAHIA'S ENTERPRISE DISTRIBUTIONS LICENSING - IMPORTANT INFORMATION
+ * ==========================================================================================
+ *
+ *     Copyright (C) 2002-2021 Jahia Solutions Group. All rights reserved.
+ *
+ *     This file is part of a Jahia's Enterprise Distribution.
+ *
+ *     Jahia's Enterprise Distributions must be used in accordance with the terms
+ *     contained in the Jahia Solutions Group Terms &amp; Conditions as well as
+ *     the Jahia Sustainable Enterprise License (JSEL).
+ *
+ *     For questions regarding licensing, support, production usage...
+ *     please contact our team at sales@jahia.com or go to http://www.jahia.com/license.
+ *
+ * ==========================================================================================
+ */
+package org.jahia.modules.sitemap.utils;
+
+/**
+ * Utility class for simple conversions
+ */
+public class ConversionUtils {
+
+    public static long longVal(Long l, long defaultValue) {
```

Comment:
Maybe I'm just being picky but would it be ok to rename the function from `longVal` to `getLongValue` or `getLongVal`

Reply:
I guess I am trying to mimic the default function name of longValue() for Long box type
Personally I think it should be ok in this case but up for discussion :)

Reply:
👍🏻

Reply:
👍